docs/fedora-releases-guide/releasing-multiple-packages.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Releasing multiple packages in a single update
+sidebar_position: 7
+---
+# Releasing multiple packages in a single update
+
+It is possible to release multiple packages in a single Bodhi update using Koji sidetags.
+Packit now supports automating that. Let's take a look on how to configure it.
+
+## Simplest scenario
+
+Let's have two packages, `foo` and `bar`, that we want to release together in one update.
+The packages build independently of each other.
+
+Here are the relevant parts from Packit config of `foo`:
+
+```yaml
+downstream_package_name: foo
+
+jobs:
+  - job: koji_build
+    trigger: commit
+    sidetag_group: foo-bar-releases
+    dependents:
+      - bar
+    dist_git_branches:
+      - fedora-all
+```
+
+Here are the relevant parts from Packit config of `bar`:
+
+```yaml
+downstream_package_name: bar
+
+jobs:
+  - job: koji_build
+    trigger: commit
+    sidetag_group: foo-bar-releases
+    dist_git_branches:
+      - fedora-all
+
+  - job: bodhi_update
+    trigger: koji_build
+    sidetag_group: foo-bar-releases
+    dependencies:
+      - foo
+    dist_git_branches:
+      - fedora-all
+```
+
+Note there is only one `bodhi_update` job for both packages, we chose to configure it in `bar`.
```

Comment:
```suggestion
Note that there is only one `bodhi_update` job for both packages, we chose to configure it in `bar`.
```
Not sure, but it sounds to me more correct.

Reply:
Both are correct. But since there is another occurence of `Note that` below, let's make it consistent.